.gitignore

```diff
@@ -15,6 +15,10 @@ core/Enclave/*.o
 
 bin/
 
```

Comment:
Do not merge the compiled .o files in lib_jsoncpp like: lib_jsoncpp/json_writer.o

Reply:
It will be fix.